### ISSUE TITLE
ArgumentNullException in SetAclCommand running Install-ContainerOSImage

### ISSUE THREAD

**Issue by ericwj** [PowerShell/ContainerProvider#6]
On a cleanly installed Nanoserver TP4, connected using PS Direct, `Install-ContainerOSImage` and `Install-ContainerImage` fail with the following error. Reconnecting over WinRM to the IP-address, the `Install-ContainerOSImage` command succeeds.

```
[Nano]: PS C:\Users\Administrator\Documents> Install-ContainerImage -name NanoServer -version 10.0.10586.0
Value cannot be null.
Parameter name: SafeHandle cannot be null.
    + CategoryInfo          : NotSpecified: (:) [Set-Acl], ArgumentNullException
    + FullyQualifiedErrorId : System.ArgumentNullException,Microsoft.PowerShell.Commands.SetAclCommand
```

I'm using `ContainerProvider` version 0.5.2.

The last thing it lets me read is "Virtual Hard Disk Creation" after "Formatting Volume" etc

The image has the packages matching "IIS", "DSC", "Guest", "Containers", "ReverseForwarders" installed using the `New-NanoServerImage` command.

**Comment by Krishna-Vutukuri** [comment]
Thanks for reporting this issue. We are aware of this issue and are working to resolve this in future releases. Until then please use the workaround of using PowerShell Remoting over WinRM + Install-ContainerOSImage

**Comment by jayshah-msft** [comment]
@ericwj

This issue has been resolved in version 0.6.4.0. Please use this version to obtain the latest container images. Let us know if you face any issues.

Thank you for your patience.

Cheers,
Jay!
